pkg/audit/verifier_test.go

```diff
@@ -397,7 +398,7 @@ func TestVerifierMissingPiece(t *testing.T) {
 		report, err := audits.Verifier.Verify(ctx, stripe, nil)
 		require.NoError(t, err)
 
-		assert.Len(t, report.Successes, len(stripe.Segment.GetRemote().GetRemotePieces())-1)
+		assert.Len(t, report.Successes, origNumPieces-1)
```

Comment:
This change should be applied to all relevant tests. I've seen other tests occasionally failing because the pointer was modified.

e.g. https://build.ops.storj.io/blue/organizations/jenkins/storj/detail/PR-2404/33/tests

Reply:
If you know of a place where it is not, let me know.  I was under the impression I fixed it everywhere I needed to.